osu.Game.Tests/Visual/Ranking/TestSceneStarRatingDisplay.cs

```diff
@@ -25,8 +30,14 @@ public TestSceneStarRatingDisplay()
                     new StarRatingDisplay(new StarDifficulty(5.67, 0)),
                     new StarRatingDisplay(new StarDifficulty(6.78, 0)),
                     new StarRatingDisplay(new StarDifficulty(10.11, 0)),
+                    changingStarRating = new StarRatingDisplay(),
                 }
             };
-        }
+
+            Scheduler.AddDelayed(() =>
+            {
+                changingStarRating.Current.Value = new StarDifficulty(RNG.NextDouble(0, 10), RNG.Next());
+            }, 500, true);
+        });
```

Comment:
not sure why this is a delayed schedule inside a schedule, while this test scene could just use normal test steps

Reply:
sure.